### PR TITLE
Integrate Construction API 2.0

### PR DESCRIPTION
**Pull request by patrick-ogrady**
Related Issue: #112 
Blocking PR: https://github.com/coinbase/rosetta-sdk-go/pull/118

### Changes
- [x] remove default configuration for `construction` in `configuration`
- [x] integrate `rosetta-sdk-go/constructor`
- [x] add script to validate that example configuration files are correct
- [x] update configuration files (add back Bitcoin in future PR)

### CODE REVIEW

File: pkg/processor/coordinator_helper.go [hunk 1]
```diff
@@ -218,103 +236,129 @@ func (c *ConstructorHelper) Sign(
...
 		payload,
+		confirmationDepth,
 	)
 }
 
-// AllAddresses returns a slice of all known addresses.
-func (c *ConstructorHelper) AllAddresses(ctx context.Context) ([]string, error) {
-	return c.keyStorage.GetAllAddresses(ctx)
+// BroadcastAll attempts to broadcast all ready transactions.
+func (c *CoordinatorHelper) BroadcastAll(
```
Comment (qiwu7) [inline pkg/processor/coordinator_helper.go:345]: How does this method handle partial failures, i.e. if one broadcast fails, does the subsequent one also fails or it will continue broadcasting?

Comment (patrick-ogrady) [reply pkg/processor/coordinator_helper.go:345]: The caller defines a `BroadcastLimit` for the max number of times to attempt re-broadcast (https://github.com/coinbase/rosetta-cli/blob/04f379541ad59a15a9cbf0e8472b12cedbcc2fa7/configuration/configuration.go#L186-L188) and if broadcast failures should be ignored (https://github.com/coinbase/rosetta-cli/blob/04f379541ad59a15a9cbf0e8472b12cedbcc2fa7/configuration/configuration.go#L190-L192).

TL;DR however you want!
